lib/rules/no-url-protocols.js

```diff
@@ -2,30 +2,34 @@
 
 var helpers = require('../helpers');
 
-var isUrlRegex = /^(https?:)?\/\//;
-
-var stripQuotes = function (str) {
-  return str.substring(1, str.length - 1);
-};
+var isUrlRegex = /^(https?:)?\/\//,
+    protocolRelativeRegex = /^(https?:)\/\//;
 
 module.exports = {
   'name': 'no-url-protocols',
-  'defaults': {},
+  'defaults': {
+    'protocol-relative-urls': false
```

Comment:
I think this needs a positive or negative prefix, such as `allow-protocol-relative-urls` This would reverse the logic slightly and mean you could remove the logical operators when assigning `regexSelector` and `message` below